java-checks/src/test/files/checks/ReplaceLambdaByMethodRefCheck.java

```diff
@@ -168,3 +169,41 @@ void test(Object param) {
     bar(new D(param)::foo);
   }
 }
+
+class SONARJAVA_2795 {
```

Comment:
I'm not a huge fan of using ticket name in test cases. Can you explicitly name this class with what it tests? The ticket can be retrieved by blaming the test file, while when reading the test, it's easier to read the "why".